examples_tests/49.Radix_Sort/Create_Compute_Shader.h

```diff
@@ -0,0 +1,72 @@
+#ifndef _CREATE_COMPUTE_SHADER_H_INCLUDED__
+#define _CREATE_COMPUTE_SHADER_H_INCLUDED__
+
+#include <cstdint>
+
+#include <CReadFile.h>
+#include <COpenGLExtensionHandler.h>
+
+namespace Radix_Sort
+{
+    inline unsigned createComputeShader(const char* _src)
+    {
+        using namespace nbl;
+
+        unsigned program = video::COpenGLExtensionHandler::extGlCreateProgram();
+        unsigned cs = video::COpenGLExtensionHandler::extGlCreateShader(GL_COMPUTE_SHADER);
+
+        video::COpenGLExtensionHandler::extGlShaderSource(cs, 1, const_cast<const char**>(&_src), NULL);
+        video::COpenGLExtensionHandler::extGlCompileShader(cs);
+
+        // check for compilation errors
+        GLint success;
+        GLchar infoLog[0x200];
+        video::COpenGLExtensionHandler::extGlGetShaderiv(cs, GL_COMPILE_STATUS, &success);
+        if (!success)
+        {
+            video::COpenGLExtensionHandler::extGlGetShaderInfoLog(cs, sizeof(infoLog), nullptr, infoLog);
+            os::Printer::log("CS COMPILATION ERROR:\n", infoLog, ELL_ERROR);
+            video::COpenGLExtensionHandler::extGlDeleteShader(cs);
+            video::COpenGLExtensionHandler::extGlDeleteProgram(program);
+            return 0;
+        }
+
+        video::COpenGLExtensionHandler::extGlAttachShader(program, cs);
+        video::COpenGLExtensionHandler::extGlLinkProgram(program);
+
+        //check linking errors
+        success = 0;
+        video::COpenGLExtensionHandler::extGlGetProgramiv(program, GL_LINK_STATUS, &success);
+        if (success == GL_FALSE)
+        {
+            video::COpenGLExtensionHandler::extGlGetProgramInfoLog(program, sizeof(infoLog), nullptr, infoLog);
+            os::Printer::log("CS LINK ERROR:\n", infoLog, ELL_ERROR);
+            video::COpenGLExtensionHandler::extGlDeleteShader(cs);
+            video::COpenGLExtensionHandler::extGlDeleteProgram(program);
+            return 0;
+        }
+
+        return program;
+    }
```

Comment:
thats not how to do it aat all

Reply:
right so, this file is gonna to be deleted? I just found it out at one of examples

Reply:
yeah, must be some extremely old code, see example 02 instead

Reply:
@Crisspl thanks Criss, I will check it, good night